platforms/xamarin.ios/System.Runtime.Serialization.csproj

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
+    <AssemblyOriginatorKeyFile>../silverlight.pub</AssemblyOriginatorKeyFile>
```

Comment:
I tried it, but to make it work I'd need to add a PropertyGroup in the .csprojs _before_ the import of dir.props otherwise settings `UseSilverlightKey=true` in the csproj doesn't have any effect.

Should I do it that way or is there a better solution?